src/hooks/useResetAccountState.ts

```diff
@@ -13,9 +12,9 @@ export default function useResetAccountState() {
     const p0 = dispatch(explorerClearState());
     const p1 = dispatch(dataResetState());
     const p2 = dispatch(requestsResetState());
-    const p3 = dispatch(uniswapResetState());
```

Comment:
removed this bc it never worked to begin with. this reset the uniswap state, but not local storage, which would immediately overwrite the uniswap state